hack/deploy-karmada-agent.sh

```diff
@@ -59,17 +59,7 @@ fi
 export KUBECONFIG="${MEMBER_CLUSTER_KUBECONFIG}" # switch to member cluster
 kubectl config use-context "${MEMBER_CLUSTER_NAME}"
 
-AGENT_IMAGE_PULL_POLICY="Always" # default is 'always' for standalone member cluster to try to use the latest version
-# load image if member cluster created by kind (if kubeconfig clusters' name contains kind-xxx)
-if kubectl config get-clusters | grep "kind-${MEMBER_CLUSTER_NAME}"
-then
-  # make agent image
-  export VERSION="latest"
-  export REGISTRY="swr.ap-southeast-1.myhuaweicloud.com/karmada"
-  make image-karmada-agent --directory="${REPO_ROOT}"
-  kind load docker-image "${REGISTRY}/karmada-agent:${VERSION}" --name="${MEMBER_CLUSTER_NAME}"
```

Comment:
1. `kind load` is needed, you cannot delete it.
2. Shall we set `AGENT_IMAGE_PULL_POLICY="IfNotPresent"` always？

Reply:
In `local-up-karmada-agent.sh`  line 157 load the image `kind load docker-image "${REGISTRY}/karmada-agent:${VERSION}" --name="${PULL_MODE_CLUSTER_NAME}"`

Reply:
May be deploy agent in cluster created by kind need `AGENT_IMAGE_PULL_POLICY="IfNotPresent"`. Let me check this.